app/src/main/java/fr/free/nrw/commons/customselector/helper/OnSwipeTouchListener.kt

```diff
@@ -32,7 +32,7 @@ open class OnSwipeTouchListener(context: Context?) : View.OnTouchListener {
 
     inner class GestureListener : GestureDetector.SimpleOnGestureListener() {
 
-        override fun onDown(e: MotionEvent?): Boolean {
+        override fun onDown(e: MotionEvent): Boolean {
```

Comment:
Changed parameter type from e: MotionEvent? to MotionEvent.

Fixes error: `'onDown' overrides nothing`